builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -94,6 +124,15 @@ func resourceAwsS3BucketRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	// Add website_endpoint as an output
+	endpoint, err := websiteEndpoint(s3conn, d)
+	if err != nil {
+		return err
+	}
+	if err := d.Set("website_endpoint", endpoint); err != nil {
+		return err
+	}
```

Comment:
This doesn't seem to set `website_endpoint` to `""` if you disable website support on an existing bucket. How do I trigger other resources that depend on that attribute to be updated?